### ISSUE TITLE
Updating homepage

### ISSUE THREAD

**Issue by swati2603** [LWM-Community/lwm_Hacktober_repo_1#7]
On the homepage, images and content are mixed in some places.

**Comment by ikramulkayes** [comment]
Yes, they are. You can fix them or can add more relevant pictures. 
Yes, you have been assigned.

**Comment by yas-sisodia** [comment]
@ikramulkayes Please assign this task to me i can solve your issue
